lib/middleware/resolve-project-version-updates.js

```diff
@@ -0,0 +1,13 @@
+const { get, noop } = require('lodash');
+const resolve = require('../hooks/resolve');
+
+module.exports = settings => (req, res, next) => {
+  const model = get(req.body, 'model');
+  const action = get(req.body, 'action');
+  if (model === 'projectVersion' && action === 'update') {
+    return resolve(settings)({ data: req.body, update: noop })
```

Comment:
We'll want to move this outside of the middleware invocation, so the hook is initialised with the settings exactly once.

```js
module.exports = settings => {
  const hook = resolve(settings);
  return (req, res, next) => {
    ...
  }
}
```

Reply:
yes, good point